addons/iptables.yml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
PR #1040 will address that issue. If it gets merged before this one, we should definitely use an `include`. Good thoughts as always @eirslett :)